concepts/regular-expressions/.meta/config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "blurb": "A Regular expression is a powerful way of manipulating and searching through data with JavaScript. Learn common implementations and help building a chatbot along the way.",
```

Comment:
- This is the blurb for the concept. It should not include a reference to the exercise. You can remove the second sentence.
- "Regular" should be "regular"